pocket_world_application/src/main/java/org/exschool/pocketworld/model/Building.java

```diff
@@ -0,0 +1,67 @@
+package org.exschool.pocketworld.model;
+
+import java.util.List;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+
+@Entity
+public class Building {
+	@ManyToOne
+	@JoinColumn(name="city")
+	private City city;
+	@Id
+	private BuildingsType type;
```

Comment:
you should use embedded id here http://stackoverflow.com/questions/3585034/how-to-map-a-composite-key-with-hibernate